2021-10-01 15:40:32,414 INFO    MainThread:116866 [wandb_setup.py:_flush():69] setting env: {}
2021-10-01 15:40:32,414 INFO    MainThread:116866 [wandb_setup.py:_flush():69] setting login settings: {}
2021-10-01 15:40:32,414 INFO    MainThread:116866 [wandb_init.py:_log_setup():342] Logging user logs to /home/indika/Programming/Projects/Python/Artifical-Intelligence/PyTorch/CNN/Waste-Classification/wandb/run-20211001_154032-3j9sqii8/logs/debug.log
2021-10-01 15:40:32,414 INFO    MainThread:116866 [wandb_init.py:_log_setup():343] Logging internal logs to /home/indika/Programming/Projects/Python/Artifical-Intelligence/PyTorch/CNN/Waste-Classification/wandb/run-20211001_154032-3j9sqii8/logs/debug-internal.log
2021-10-01 15:40:32,415 INFO    MainThread:116866 [wandb_init.py:_jupyter_setup():294] configuring jupyter hooks <wandb.sdk.wandb_init._WandbInit object at 0x7f9e0c574a10>
2021-10-01 15:40:32,417 INFO    MainThread:116866 [wandb_init.py:init():375] calling init triggers
2021-10-01 15:40:32,417 INFO    MainThread:116866 [wandb_init.py:init():382] wandb.init called with sweep_config: {}
config: {}
2021-10-01 15:40:32,417 INFO    MainThread:116866 [wandb_init.py:init():397] re-initializing run, found existing run on stack: 3qch5x4p
2021-10-01 15:40:32,418 INFO    MainThread:116866 [wandb_run.py:finish():1271] finishing run ranuga-d/Waste-Classification/3qch5x4p
2021-10-01 15:40:32,418 INFO    MainThread:116866 [wandb_run.py:_atexit_cleanup():1571] got exitcode: 0
2021-10-01 15:40:32,419 INFO    MainThread:116866 [wandb_run.py:_restore():1543] restore
2021-10-01 15:40:32,937 INFO    MainThread:116866 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 720
  total_bytes: 720
}

2021-10-01 15:40:33,042 INFO    MainThread:116866 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 720
  total_bytes: 720
}

2021-10-01 15:40:34,117 INFO    MainThread:116866 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 720
  total_bytes: 720
}

2021-10-01 15:40:34,915 INFO    MainThread:116866 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 720
  total_bytes: 24115
}

2021-10-01 15:40:35,018 INFO    MainThread:116866 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 720
  total_bytes: 24115
}

2021-10-01 15:40:35,119 INFO    MainThread:116866 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 720
  total_bytes: 24115
}

2021-10-01 15:40:35,221 INFO    MainThread:116866 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 720
  total_bytes: 24115
}

2021-10-01 15:40:35,323 INFO    MainThread:116866 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 720
  total_bytes: 24115
}

2021-10-01 15:40:35,424 INFO    MainThread:116866 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 720
  total_bytes: 24115
}

2021-10-01 15:40:35,526 INFO    MainThread:116866 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 720
  total_bytes: 24115
}

2021-10-01 15:40:35,627 INFO    MainThread:116866 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 720
  total_bytes: 24115
}

2021-10-01 15:40:35,729 INFO    MainThread:116866 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 720
  total_bytes: 24115
}

2021-10-01 15:40:35,830 INFO    MainThread:116866 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 720
  total_bytes: 24115
}

2021-10-01 15:40:35,931 INFO    MainThread:116866 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 720
  total_bytes: 24115
}

2021-10-01 15:40:36,033 INFO    MainThread:116866 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 720
  total_bytes: 24115
}

2021-10-01 15:40:36,134 INFO    MainThread:116866 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 720
  total_bytes: 24115
}

2021-10-01 15:40:36,236 INFO    MainThread:116866 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 720
  total_bytes: 24115
}

2021-10-01 15:40:36,337 INFO    MainThread:116866 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 720
  total_bytes: 24115
}

2021-10-01 15:40:36,438 INFO    MainThread:116866 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 720
  total_bytes: 24115
}

2021-10-01 15:40:36,540 INFO    MainThread:116866 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 720
  total_bytes: 24115
}

2021-10-01 15:40:36,641 INFO    MainThread:116866 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 24115
  total_bytes: 24115
}

2021-10-01 15:40:36,744 INFO    MainThread:116866 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 24115
  total_bytes: 24115
}

2021-10-01 15:40:36,845 INFO    MainThread:116866 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 24115
  total_bytes: 24115
}

2021-10-01 15:40:36,947 INFO    MainThread:116866 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 24115
  total_bytes: 24115
}

2021-10-01 15:40:37,048 INFO    MainThread:116866 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 24115
  total_bytes: 24115
}

2021-10-01 15:40:37,150 INFO    MainThread:116866 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 24115
  total_bytes: 24115
}

2021-10-01 15:40:37,251 INFO    MainThread:116866 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 24115
  total_bytes: 24115
}

2021-10-01 15:40:37,352 INFO    MainThread:116866 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 24115
  total_bytes: 24115
}

2021-10-01 15:40:37,453 INFO    MainThread:116866 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 24115
  total_bytes: 24115
}

2021-10-01 15:40:37,555 INFO    MainThread:116866 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 24115
  total_bytes: 24115
}

2021-10-01 15:40:37,656 INFO    MainThread:116866 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 24115
  total_bytes: 24115
}

2021-10-01 15:40:37,758 INFO    MainThread:116866 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 24115
  total_bytes: 24115
}

2021-10-01 15:40:38,284 INFO    MainThread:116866 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 24115
  total_bytes: 24115
}

2021-10-01 15:40:38,385 INFO    MainThread:116866 [wandb_run.py:_wait_for_finish():1693] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 24115
  total_bytes: 24115
}

2021-10-01 15:40:39,481 INFO    MainThread:116866 [wandb_run.py:_show_files():1915] logging synced files
2021-10-01 15:40:39,487 INFO    MainThread:116866 [wandb_init.py:init():424] starting backend
2021-10-01 15:40:39,487 INFO    MainThread:116866 [backend.py:_multiprocessing_setup():72] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2021-10-01 15:40:39,490 INFO    MainThread:116866 [backend.py:ensure_launched():135] starting backend process...
2021-10-01 15:40:39,536 INFO    MainThread:116866 [backend.py:ensure_launched():140] started backend process with pid: 127762
2021-10-01 15:40:39,537 INFO    MainThread:116866 [wandb_init.py:init():429] backend started and connected
2021-10-01 15:40:39,547 INFO    MainThread:116866 [wandb_run.py:_label_probe_notebook():815] probe notebook
2021-10-01 15:40:39,548 INFO    MainThread:116866 [wandb_init.py:init():477] updated telemetry
2021-10-01 15:40:39,548 INFO    MainThread:116866 [wandb_init.py:init():500] communicating current version
2021-10-01 15:40:40,540 INFO    MainThread:116866 [wandb_init.py:init():505] got version response upgrade_message: "wandb version 0.12.3 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2021-10-01 15:40:40,541 INFO    MainThread:116866 [wandb_init.py:init():513] communicating run to backend with 30 second timeout
2021-10-01 15:40:41,212 INFO    MainThread:116866 [wandb_init.py:init():540] starting run threads in backend
2021-10-01 15:40:46,218 INFO    MainThread:116866 [wandb_run.py:_console_start():1601] atexit reg
2021-10-01 15:40:46,218 INFO    MainThread:116866 [wandb_run.py:_redirect():1475] redirect: SettingsConsole.WRAP
2021-10-01 15:40:46,219 INFO    MainThread:116866 [wandb_run.py:_redirect():1512] Wrapping output streams.
2021-10-01 15:40:46,219 INFO    MainThread:116866 [wandb_run.py:_redirect():1536] Redirects installed.
2021-10-01 15:40:46,219 INFO    MainThread:116866 [wandb_init.py:init():565] run started, returning control to user process
2021-10-01 15:40:46,250 INFO    MainThread:116866 [wandb_init.py:_pause_backend():256] pausing backend
2021-10-01 15:40:46,250 INFO    MainThread:116866 [jupyter.py:save_ipynb():298] not saving jupyter notebook
2021-10-01 15:41:00,951 INFO    MainThread:116866 [wandb_init.py:_resume_backend():265] resuming backend
2021-10-01 15:41:00,952 INFO    MainThread:116866 [wandb_init.py:_pause_backend():256] pausing backend
2021-10-01 15:41:00,953 INFO    MainThread:116866 [jupyter.py:save_ipynb():298] not saving jupyter notebook
2021-10-01 15:41:01,103 INFO    MainThread:116866 [wandb_init.py:_resume_backend():265] resuming backend
2021-10-01 15:41:05,087 INFO    MainThread:116866 [wandb_init.py:_pause_backend():256] pausing backend
2021-10-01 15:41:05,087 INFO    MainThread:116866 [jupyter.py:save_ipynb():298] not saving jupyter notebook
2021-10-01 15:41:05,092 INFO    MainThread:116866 [wandb_init.py:_resume_backend():265] resuming backend
2021-10-01 15:41:05,317 INFO    MainThread:116866 [wandb_setup.py:_flush():69] setting env: {}
2021-10-01 15:41:05,317 INFO    MainThread:116866 [wandb_setup.py:_flush():69] setting login settings: {}
2021-10-01 15:41:05,317 INFO    MainThread:116866 [wandb_init.py:_log_setup():342] Logging user logs to /home/indika/Programming/Projects/Python/Artifical-Intelligence/PyTorch/CNN/Waste-Classification/wandb/run-20211001_154105-2r5bqbqo/logs/debug.log
2021-10-01 15:41:05,317 INFO    MainThread:116866 [wandb_init.py:_log_setup():343] Logging internal logs to /home/indika/Programming/Projects/Python/Artifical-Intelligence/PyTorch/CNN/Waste-Classification/wandb/run-20211001_154105-2r5bqbqo/logs/debug-internal.log
2021-10-01 15:41:05,317 INFO    MainThread:116866 [wandb_init.py:init():375] calling init triggers
2021-10-01 15:41:05,317 INFO    MainThread:116866 [wandb_init.py:init():382] wandb.init called with sweep_config: {}
config: {}
2021-10-01 15:41:05,317 INFO    MainThread:116866 [wandb_init.py:init():397] re-initializing run, found existing run on stack: 3j9sqii8
2021-10-01 15:41:05,318 INFO    MainThread:116866 [wandb_run.py:finish():1271] finishing run ranuga-d/Waste-Classification/3j9sqii8
